### ISSUE TITLE
The accuracy of baseline CNN model remain very low.

### ISSUE THREAD

**Issue by Huhanlin** [JaesungBae/SCR-CapsNet#2]
After I run the code:
python main.py --model='CNN' --ex_name='ref_2015is_cnn' --is_training='TRAIN' --model_size_info 21 8 94 1 1 2 3 6 4 94 1 1 1 1 32

Here's the result
cat KWS/save/CNN/ref_2015is_cnn_clean_clean/log.csv 
epoch,acc,loss,val_acc,val_loss
0,0.0352528969621,3.3952801742,0.0347160929751,3.38853546625
1,0.0358401190103,3.39308315977,0.0347160929751,3.38782089975
2,0.0346265267773,3.39311181181,0.0347160929751,3.38773612234
3,0.0336673974319,3.39305867142,0.0338334804354,3.38793892052
4,0.0346265267773,3.39307627729,0.0385407472786,3.38806093325
5,0.0359967115565,3.39306007393,0.038687849374,3.38764988573
6,0.0349788600063,3.39303597402,0.0385407472786,3.3879062811
7,0.0343720638898,3.39304099634,0.0364813180479,3.38780447545
8,0.0352137488256,3.39308122203,0.0376581347455,3.38781898111
9,0.0341958972753,3.39309233033,0.036187113857,3.38771080178

Which is really confusing me, could u please give me some help?

**Comment by Huhanlin** [comment]
Sry about the question, it's the learning rate... After I adjust the learning rate to 0.00001, it works out well.
